docs/user_docs/upgrade/upgrade with helm/upgrade-kubeblocks-to-0.9.1.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Upgrade to KubeBlocks v0.9.1
+description: Upgrade to KubeBlocks v0.9.1, operation, tips and notes
+keywords: [upgrade, 0.9.1]
+sidebar_position: 1
+sidebar_label: Upgrade to KubeBlocks v0.9.1
+---
+
+# Upgrade to KubeBlocks v0.9.1
+
+:::note
+
+Execute `helm -n kb-system list | grep kubeblocks` to check the current KubeBlocks version you are running before upgrading KubeBlocks.
+
+:::
+
+## Compatibility
+
+KubeBlocks v0.9.1 is compatible with KubeBlocks v0.8 APIs, but compatibility with APIs from versions prior to v0.8 is not guaranteed. If you are using Addons from KubeBlocks v0.7 or earlier (v0.7., v0.6., etc), DO [upgrade KubeBlocks and all Addons to v0.8 first](./upgrade-kubeblocks-to-0.8.md) to ensure service availability before upgrading to v0.9.
+
+If you are upgrading from v0.8 to v0.9, it's recommended to enable webhook to ensure the availability.
+
+## Upgrade from KubeBlocks v0.9.0
+
+1. Set keepAddons.
```

Comment:
How about "Preserve Addon resources during upgrade"?

Reply:
> How about "Preserve Addon resources during upgrade"?

Great suggestion! I've added this sentence as the purpose of step 1 in the section "Upgrade from KubeBlocks v0.8" and adjusted related contents in the upgrade guide for v0.9.0.
For the section "Upgrade from v0.9.0", the description is a little different since `keepAddon` is set as `true` by default in Helm. The user just needs to check its value. But `crd.enabled` should be set as `false` to avoid automatic CRD installation.

Reply:
`crd.enabled` can be set when upgrade kubeblocks. For preserve addon resource, recommend to run follow command:

```
kubectl get addons -o name | xargs -I {} kubectl patch {} --type='json' -p='[{"op": "add", "path": "/metadata/annotations/helm.sh~1resource-policy", "value": "keep"}]'```